deptofdefense/move.mil
# Add content to Moving Tips page

## Checklist

I have…


## Summary of Changes

This pull request resolves #68 by adding content to the Moving Tips page. It also refactors the tab UI JavaScript, providing the same functionality in a slightly more organized pattern.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout moving-tips-page`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails server`, and
1. load up http://localhost:3000/moving-guide/tips in your Web browser of choice.

## Screenshots

### Narrow viewport

<img width="447" alt="narrow" src="https://user-images.githubusercontent.com/27780860/30227530-0822d246-94a8-11e7-9967-04feb7be31fa.png">

### Wide viewport

<img width="1347" alt="wide" src="https://user-images.githubusercontent.com/27780860/30227542-0c16a94a-94a8-11e7-8972-f32a14334ad8.png">
